lib/migrations/migrate/stub/ts.stub

```diff
@@ -1,8 +1,8 @@
-import * as Knex from "knex";
+import { Knex } from "knex";
 
 <% if (d.tableName) { %>
 export async function up(knex: Knex): Promise<Knex.SchemaBuilder> {
-    return knex.schema.createTable("<%= d.tableName %>", (t: Knex.AlterTableBuilder) => {
+    return knex.schema.createTable("<%= d.tableName %>", (t) => {
```

Comment:
Is there a reason why `Knex.AlterTableBuilder` was removed? Is it redundant now?

Reply:
Yes, it's redundant, TypeScript infers the type which is actually `Knex.CreateTableBuilder` not `Knex.AlterTableBuilder`.